views/editpost.ejs

```diff
@@ -0,0 +1,131 @@
+    $(document).ready(function() {
+        if (urlParams.get('tm').length != 0) {
+            if (urlParams.get('ts') == 'true') {
+                $('#toasttitle').text("Success!");
+            } else {
+                $('#toasttitle').text("Error!");
+                $('.toast-body').text(urlParams.get('tm'));
+                $('.toast').toast('show');
+            }
+        }
+    });
+</script>
+
+<!--The big container-->
+<div class="container">
+    <!-- alert saying that the user does not have access to this page-->
+
+    <% if (typeof(profane) !=='undefined' ) {%>
+        <div class="alert alert-danger" role="alert">
+            <!--logo svg filled cross circle-->
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-x-circle" viewBox="0 0 16 16">
+                <path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z" />
+                <path
+                        d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z" />
+            </svg> Request Rejected<br> Please keep the content appropriate and courteous.
+            <!--Internal Server Error-->
+        </div>
+    <% }%>
+
+
+    <div class="row" style="justify-content: center;">
+
+        <div class="col-md-8 col-md-offset-2">
+
+            <h1>Edit post</h1>
+
+            <form action="/editpost" method="POST">
+
+                <div class="has-error">
+                    <label for="slug">User</label>
+                    <input type="text" class="form-control" name="slug" disabled value="<%- email -%>" />
+                </div>
+
+                <div class="form-group">
+                    <label for="title">Title <span class="require">*</span></label>
+                    <input type="text" class="form-control" value="<%- post.subject -%>" name="subject" />
+                    <input type="text" class="form-control" name="camefrom" value="webpage" hidden />
+                </div>  
+
+                <div class="form-group">
+                    <label for="description">Description <span class="require">*</span></label>
+                    <textarea id = "editor-container" rows="5" class="form-control" name="body"><%- post.body -%></textarea>
+                </div>
+
+                <div class="form-group">
+                    <p><span class="require">*</span> - required fields</p>
+                </div>
+
+                <div class="form-group">
+                    <button type="submit" class="btn btn-primary">
+                        Save
+                    </button>
+                    <a href="/forum" class="btn btn-default" role="button" aria-pressed="true">Cancel</a>
+                    </button>
+                </div>
+
+            </form>
+        </div>
+    </div>
+    <script>
+    let toolbarOptions = [
```

Comment:
Remove unnecessary code. The variable toolbarOptions is not being used, hence it should be removed.